vision-zero

```diff
@@ -0,0 +1,330 @@
+#!env python
+
+import time
+import argparse
+import subprocess
+import os
+import shtab
+from datetime import date
+import pprint
+from dotenv import load_dotenv
+
+load_dotenv()
+
+def doCommand(args):
+    if args.command == "build":
+        buildTools(args)
+    elif args.command == "db-up":
+        dbUp(args)
+    elif args.command == "db-down":
+        dbDown(args)
+    elif args.command == "graphql-engine-up":
+        dbUp(args)
+        graphqlEngineUp(args)
+    elif args.command == "graphql-engine-down":
+        graphqlEngineDown(args)
+    elif args.command == "vze-up":
+        vzeUp(args)
+    elif args.command == "vze-down":
+        vzeDown(args)
+    elif args.command == "vzv-up":
+        vzvUp(args)
+    elif args.command == "vzv-down":
+        vzvDown(args)
+    elif args.command == "psql":
+        psql(args)
+    elif args.command == "tools-shell":
+        dbUp(args)
+        toolsShell(args)
+    elif args.command == "stop":
+        stop(args)
+    elif args.command == "replicate-db":
+        dbUp(args)
+        replicateDb(args)
+    elif args.command == "remove-snapshots":
+        removeSnapshots(args)
+
+
+## function to list the files in a directory and delete them
+def removeSnapshots(args):
```

Comment:
I also had an issue with running function with the command. I added my output below, but it works fine if I run from the command line without `subprocess`. Another tester could help see if this is a problem with my environment.

`FileNotFoundError: [Errno 2] No such file or directory: 'rm -rvf atd-vzd/snapshots/*.sql'`

Reply:
Another awesome catch, thank you Mike. I have no idea why this was working for me previously, but it sure wasn't this morning and it was exhibiting the same error you described.

I scraped the `rm` invocation and am using the `os` library now with a loop. Feels more pythonic, maybe?